### PR TITLE
data: gurjeetsinghvirdee

### PR DESCRIPTION
**Pull request by gurjeetsinghvirdee**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

-  Add utkarsh006 testimonial in my profile
-  update my discord profile link
-  socials & tags added

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3052"></a>

### CODE REVIEW

File: data/gurjeetsinghvirdee.json [hunk 1]
```diff
@@ -21,7 +33,7 @@
     },
     {
       "name": "Add me on Discord",
-      "url": "Guri#6145",
+      "url": "https://discordapp.com/users/916597112882495510",
```
Comment (krshkun) [inline data/gurjeetsinghvirdee.json:36]: This link does not work for me, but I do not think that impacts this PR, please look into this in the future, thanks

**Review by krshkun — APPROVED** [review]

File: data/gurjeetsinghvirdee.json (hunk 1 above)
Comment (gurjeetsinghvirdee) [reply data/gurjeetsinghvirdee.json:36]: Thankyou, I will remove this

File: data/gurjeetsinghvirdee.json [hunk 2]
```diff
@@ -20,8 +32,7 @@
       "icon": "FaTwitter"
     },
     {
-      "name": "Add me on Discord",
-      "url": "Guri#6145",
+      "name": "Add me: Guri#6145",
       "icon": "FaDiscord"
     }
```
Comment (krshkun) [inline data/gurjeetsinghvirdee.json:37]: I'm not sure if it will work without the url

**Review by krshkun — CHANGES_REQUESTED** [review]

File: data/gurjeetsinghvirdee.json (hunk 2 above)
Comment (gurjeetsinghvirdee) [reply data/gurjeetsinghvirdee.json]: for now, removing this
I will work on this later

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**